src/components/Dashboard.tsx

```diff
@@ -27,7 +27,7 @@ const useStyles = createStyles(theme => ({
     width: "100%"
   },
   projectCodingChart: {
-    height: "400px",
+    // minHeight: "400px",
```

Comment:
Remove this commented line

Reply:
Oh yeah. I forgot to remove that as I left it there for testing purposes. I had some trouble with the chart scaling.